deps/parser/parser.jl

```diff
@@ -0,0 +1,184 @@
+using JSON
+
+typestring = ""
+undotted_typestring = typestring[1:length(typestring)-1]
```

Comment:
what is typestring intended for? future namespacing?
If You need to get rid of the dot at the end use `strip(typestring, ".")`

Reply:
Yep. We have not yet resolved the internal module. It was previously used in different places, I guess we can actually remove it now.